README.md

```diff
@@ -189,6 +189,16 @@ One-step installation scripts are provided for the dependencies' installation. P
 	```bash
 	ros2 run dynamic_vino_sample image_object_client ~/Pictures/car.png
 	```
+* run face detection service sample code input from Image  
+  Run image processing service:
+	```bash
+	ros2 run dynamic_vino_sample image_people_server "image_people_server.yaml"
+	```
+  Run example application with an absolute path of an image on another console:
+	```bash
+	ros2 run dynamic_vino_sample image_people_client ~/Pictures/face.png
```

Comment:
@chaoli2 we have already archive one face sample picture in <repo-home>/data/images folder, it would be better to use it.

Reply:
@chaoli2 I re-wrote your solution about service I/F support in [PR60](https://github.com/intel/ros2_openvino_toolkit/pull/60), because I think the new implementation would better re-use the existed inference logic and better decouple the dependency between service interface and other normal interfaces' support. Please have a review.  I'd like to close this PR without merging, if you have no more comments.

Reply:
I agree to close this PR. My comments are added in another thread.